news/_posts/core-weekly/2018-12-20-coreweekly-week-50-2018.md

```diff
@@ -0,0 +1,207 @@
+---
+layout: post
+title:  "PrestaShop Core Weekly - Week 50 of 2018"
+subtitle: "An inside look at the PrestaShop codebase"
+date:   2018-12-21 16:15:00
+authors: [ AntoineThomas ]
+icon: icon-calendar
+tags:
+ - core-weekly
+---
+
+This edition of the Core Weekly report highlights changes in PrestaShop's core codebase during the last week, from Monday 10th to Sunday 16th of December 2018.
+
+![Core Weekly banner](/assets/images/2018/12/banner-core-weekly.jpg)
+
+
+## General messages
+
+Dear Developers,
+
+Did you notice the new banner on the Core Weekly? It now matches the new theme of prestashop.com ;-)
+
+Also, last week, a few records have been broken on PrestaShop project's repositories on GitHub, in a week in 2018.
```

Comment:
`in a week in 2018` looks strange to me 🤔 
I dont know how to say it better though

Reply:
yeah, me neither

Reply:
`Also, last week, a few weekly records have been broken on PrestaShop project's repositories on GitHub, for this year 2018.` ?

Reply:
what do you think of:

> Also, last week, a few 2018 records have been broken on PrestaShop project's repositories on GitHub: